README.md

```diff
@@ -1,52 +1,101 @@
+![character infographic](./characterGraphic.png)
+
 # Extra Roles
 A BepInEx mod for Among Us that adds 4 new roles into the game.
 
-![character infographic](./characterGraphic.png)
+- [Medic](#medic)
+- [Officer](#officer)
+- [Engineer](#engineer)
+- [Joker](#joker)
 
 # Notice
 This mod will work on Innersloth servers, but everybody in the lobby has to have the same version of the mod. For help with installing the mod, getting it to work, or fixing an issue, join the [Discord](https://discord.gg/j2MVs4r6cc).
 
-**This mod cannot be installed on Android/IOS/Epic Games**
+[![Discord](https://discord.com/assets/e4923594e694a21542a489471ecffa50.svg)](https://discord.gg/j2MVs4r6cc)
+
+**This mod cannot be installed on Android/iOS/Epic Games/Microsoft Store**
```

Comment:
I would move that text above the image. The focus lies too much on the large image here.

Reply:
Solved, thanks for letting me know, I didn't pay much attention because I focused more on making it more beautiful by editing and adding more stuff